Mathlib/CategoryTheory/Limits/Shapes/Pullback/HasPullback.lean

```diff
@@ -138,6 +144,16 @@ abbrev pushout.desc {W X Y Z : C} {f : X ⟶ Y} {g : X ⟶ Z} [HasPushout f g] (
   colimit.desc _ (PushoutCocone.mk h k w)
 #align category_theory.limits.pushout.desc CategoryTheory.Limits.pushout.desc
 
+/-- The cone associated to a pullback is a limit cone. -/
+abbrev pullback.isLimit {X Y Z : C} (f : X ⟶ Z) (g : Y ⟶ Z) [HasPullback f g] :
```

Comment:
Maybe call this `pullbackIsPullback`? Compare with [docs#CategoryTheory.Limits.productIsProduct](https://leanprover-community.github.io/mathlib4_docs/Mathlib/CategoryTheory/Limits/Shapes/Products.html#CategoryTheory.Limits.productIsProduct)

Reply:
There is actually already a `pullbackIsPullback`, which says that the pullback cone constructed from `pullback f g` using `PullbackCone.mk` is a limit cone (and this seems similar to what happens in [docs#CategoryTheory.Limits.productIsProduct](https://leanprover-community.github.io/mathlib4_docs/Mathlib/CategoryTheory/Limits/Shapes/Products.html#CategoryTheory.Limits.productIsProduct)).